navigator/src/epub/frame/FrameManager.ts

```diff
@@ -77,10 +77,19 @@ export default class FrameManager {
         else this.comms = new FrameComms(this.frame.contentWindow!, this.source);
         return new Promise((res, _) => {
             this.comms?.send("focus", undefined, () => {
-                this.frame.style.removeProperty("visibility");
-                this.frame.style.removeProperty("opacity");
-                this.frame.style.removeProperty("pointer-events");
-                res();
+                if(atProgress && atProgress > 0) {
```

Comment:
`undefined > 0 === false`, so if the compiler will allow it, this could just be:

 ```suggestion
                if(atProgress > 0) {
```

Reply:
![image](https://user-images.githubusercontent.com/13407784/198420763-2714f22e-e717-4816-bb85-974094acf3b4.png)
The compiler doesn't like it, I'll leave it for now